scanners/ios/workflow.go

```diff
@@ -167,14 +219,22 @@ func addSharedTeardownSteps(workflow models.WorkflowID, params workflowSetupPara
 	params.configBuilder.AppendStepListItemsTo(workflow, steps.DefaultDeployStepList()...)
 }
 
-func addDescription(projectType XcodeProjectType, workflow models.WorkflowID, configBuilder *models.ConfigBuilderModel, description string) {
-	if projectType != XcodeProjectTypeIOS {
+func addDescription(workflow models.WorkflowID, configBuilder *models.ConfigBuilderModel, description string) {
+	if description == "" {
 		return
 	}
 
 	configBuilder.SetWorkflowDescriptionTo(workflow, description)
 }
 
+func addSummary(workflow models.WorkflowID, configBuilder *models.ConfigBuilderModel, summary string) {
+	if summary == "" {
+		return
+	}
```

Comment:
Does this empty string check needed? The same for the `addDescription` method too. 

In the old `addDescription` method implementation it was guarding against the macOS project types. I can see that  the builder is generated here:
https://github.com/bitrise-io/bitrise-init/blob/02e510ae4f764d344d615a24b87634e9ff5e5ecc/scanners/ios/utility.go#L602

And it has an empty summary and description when it is created. And also a separate builder is created when invoked from the macOS scanner. So to it seems like there will not be a possibility when the builder already has a summary and we could accidentally override it with an empty string.

Reply:
Updated in https://github.com/bitrise-io/bitrise-init/pull/265/commits/22bdb98cc509306a312db9d1426f9928053d6b9c